domains/yekyawswaraung.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "yksa",
+        "email": "yekyawswaraung@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://yksa.github.io/"
```

CNAMEs cannot contain https or /
```suggestion
        "CNAME": "yksa.github.io"
```